Resources/Inconsistencies.tex

```diff
@@ -0,0 +1,88 @@
+\documentclass[11pt,fleqn]{article}
+
+\newcommand{\indentpar}{\phantom{=}}
+\newcommand{\bli}{\begin{itemize}}
+\newcommand{\eli}{\end{itemize}}
+
+\begin{document}
+
+    %% Title Page %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    \begin{center}
+
+        {\large \textbf{VDispl}}\\[2mm]
+        {\large \textbf{Soil Settlement Analysis Software}}\\[2mm]
+        {\huge \textbf{Inconsistencies in Settlements Textbook and FORTRAN Code}}\\[6mm]
```

Comment:
The `\large` and `\huge` text throws off the title spacing.  You don't need them.  (If you really want the larger text, you could look to see if there is a LaTeX class file with the title display that you like.)

Reply:
Found new Title Page template